kore/test/Test/Kore/Rewrite/Axiom/Identifier.hs

```diff
@@ -102,8 +116,36 @@ test_matchAxiomIdentifier =
             (Mock.builtinList [Mock.a, Mock.b])
             (Application Mock.concatListId)
         ]
+    , testGroup
+        "Domain Values"
+        [ test
+            "domain value"
+            ( TermLike.mkDomainValue $
+                DomainValue Mock.testSort $
+                    TermLike.mkStringLiteral "hello"
+            )
+            DV
+        , test
+            "internal string"
+            (TermLike.mkInternalString $ InternalString Mock.testSort "world")
+            DV
+        , test
+            "internal bool"
+            (TermLike.mkInternalBool $ InternalBool Mock.testSort False)
+            DV
+        , test
+            "internal bytes"
+            (TermLike.mkInternalBytes Mock.testSort ByteString.empty)
+            DV
+        , test
+            "internal int"
+            (TermLike.mkInternalInt $ InternalInt Mock.testSort 42)
+            DV
+        ]
     ]
   where
+    -- FIXME add tests for DV cases
+
```

Comment:
are you planning on adding these in this PR or leave for later?

Reply:
Oh, I actually did add those tests (last commit), should have removed the comment, too.